adulbrich/osu-fsf
# Refactor URLs for event/id & event/pastEvents

### Changes

- The new route for navigating to an event is /event/id/[ID]
  - This is a more RESTful compared to /event/ID
- New route for viewing all events for a specific category is /event/category/[EventCategory]
  - Now PastEvents and Upcoming events are all consolidated into one file. The page changes depending on the EventCategory parameter in the url


### How to test
First of all, navigate to src-web and  run `npm run dev`
- Click on all event cards on /event and see if they work
- Search for an event in the search bar and press enter.  Should navigate to /event/id/[ID]
- Click on the more button for past events and upcoming events. Should navigate to /event/category/[EventCategory].  Upcoming events should only have one event listed.  Past events should have multiple events listed.
